cmd/minikube/cmd/start.go

```diff
@@ -427,6 +427,11 @@ func validateKubernetesVersions(old *cfg.Config) string {
 	if nv == "" {
 		nv = constants.DefaultKubernetesVersion
 	}
+
+	if nv[0] != version.VersionPrefix {
```

Comment:
To avoid the bytes/string error, this should match the check in versions.go:

`if !strings.HasPrefix(nv, version.VersionPrefix) {`

Alternatively, why not just allow a version to be specified without a leading v? We know what the user wants, so instead of raising the error, we could just give it to them and make their life easier:

```go
if !strings.HasPrefix(nv, version.VersionPrefix) {
  nv = version.VersionPrefix + nv
}
```